cmd/commands/server/command_server.go

```diff
@@ -71,7 +72,16 @@ func (cmd *Command) Start() (err error) {
 		return err
 	}
 
-	cmd.vpnServer = cmd.vpnServerFactory(sessionManager)
+	stopPinger := make(chan int)
+	cmd.vpnServer = cmd.vpnServerFactory(sessionManager, func(state openvpn.State) {
```

Comment:
awkward function passing by creating inline function. Would separate anonymous function creation and pass it like a parameter, like: 

callback := func(state openvpn.State) {
..
}
cmd.vpnServerFactory(sessionManager, callback)

Reply:
Fixed